packages/calypso-data-stores/package.json

```
@@ -0,0 +1,51 @@
+{
+	"name": "@automattic/calypso-data-stores",
+	"version": "1.0.0-alpha.0",
+	"description": "Calypso Data Stores",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/calypso-data-stores"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"autocomplete.gif",
+		"dist",
+		"types",
+		"src"
+	],
+	"types": "types",
+	"scripts": {
+		"clean": "npx rimraf dist types",
+		"prepublish": "npm run clean",
+		"prepare": "transpile && tsc"
```

This package should become just:

```suggestion
		"prepare": "tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json",
```

https://github.com/Automattic/wp-calypso/blob/72fc8477fc10bcbb4848ad36cb29df49c945fdf3/packages/composite-checkout-wpcom/package.json#L9-L14

---

Ooh pure TypeScript :heart_eyes: